internal/storage/bucket_handle.go

```diff
@@ -246,7 +246,7 @@ func (b *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsRequ
 		Prefix:                   req.Prefix,
 		Projection:               getProjectionValue(req.ProjectionVal),
 		IncludeTrailingDelimiter: req.IncludeTrailingDelimiter,
-		IncludeFoldersAsPrefixes: req.IncludeFoldersAsPrefixes,
+		IncludeFoldersAsPrefixes: false,
```

Comment:
Temp change: as true value of IncludeFoldersAsPrefix doesn't work with gRPC client.

Reply:
It gives error while listing.